tasks/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Load tasks file for adding kali-dev package repo for Kali
+  ansible.builtin.include_tasks:
+    file: setup_Kali.yml
```

Comment:
Is there a particular reason you're using this form here? I only ask because there are two other task that use `ansible.builtin.include_tasks` in this file that do not use this form.

Reply:
I assume by "this form" you mean the filename `setup_Kali.yml`.  I named it that because I semi-cribbed this code from [cisagov/ansible-role-docker](https://github.com/cisagov/ansible-role-docker) where similar names are used.  I invested very little thought into it, since issue #61 exists to ensure that this foo is removed as soon as possible.

What would you like the file to be named?

Reply:
Perhaps I misunderstood your concern in the previous message.  Does commit 5b2e463 address your concern?

Reply:
Yes that commit is exactly what I meant. Sorry for the ambiguity of my review comment.